test/unit/models/classes/runner/time/AdjustmentMapTest.php

```diff
@@ -16,103 +16,142 @@
     {
-        $map = new AdjustmentMap();
-        $map->increase('testSourceId', 10);
-        $this->assertEquals(10, $map->toArray()['testSourceId'][AdjustmentMap::ACTION_INCREASE]);
-        $this->assertEquals(0, $map->toArray()['testSourceId'][AdjustmentMap::ACTION_DECREASE]);
+        $this->subject->increase('testSourceId', self::DUMMY_ADJUSTMENT_TYPE, 10);
+        $this->assertEquals(
+            10,
+            $this->subject->toArray()['testSourceId'][self::DUMMY_ADJUSTMENT_TYPE][AdjustmentMap::ACTION_INCREASE]
+        );
+        $this->assertEquals(
+            0,
+            $this->subject->toArray()['testSourceId'][self::DUMMY_ADJUSTMENT_TYPE][AdjustmentMap::ACTION_DECREASE]
+        );
     }
 
-    public function testIncrease_WhenValueExistsInTheMap_ThenSubsequentValuesAreAdded()
+    public function testIncrease_WhenValueExistsInTheMap_ThenSubsequentValuesAreAdded(): void
     {
-        $map = new AdjustmentMap();
-        $map->increase('testSourceId', 10);
-        $map->increase('testSourceId', 15);
-        $this->assertEquals(25, $map->toArray()['testSourceId'][AdjustmentMap::ACTION_INCREASE]);
+        $this->subject->increase('testSourceId', self::DUMMY_ADJUSTMENT_TYPE, 10);
+        $this->subject->increase('testSourceId', self::DUMMY_ADJUSTMENT_TYPE, 15);
+        $this->assertEquals(
+            25,
+            $this->subject->toArray()['testSourceId'][self::DUMMY_ADJUSTMENT_TYPE][AdjustmentMap::ACTION_INCREASE]
+        );
     }
 
-    public function testGet_WhenThereIsNoValuePresent_ThenZeroIsReturned()
+    public function testGet_WhenThereIsNoValuePresent_ThenZeroIsReturned(): void
     {
-        $map = new AdjustmentMap();
-        $this->assertEquals(0, $map->get('newSourceId'));
+        $this->assertEquals(0, $this->subject->get('newSourceId'));
     }
 
-    public function testGet_WhenRequested_ThenReturnValueIsCalculatedFromIncreasesAndDecreases()
+    public function testGet_WhenRequested_ThenReturnValueIsCalculatedFromIncreasesAndDecreasesOfAllTypes(): void
     {
-        $map = new AdjustmentMap();
-        $map->increase('testSourceId', 10);
-        $this->assertEquals(10, $map->get('testSourceId'));
-        $map->decrease('testSourceId', 5);
-        $this->assertEquals(5, $map->get('testSourceId'));
-        $map->decrease('testSourceId', 10);
-        $this->assertEquals(-5, $map->get('testSourceId'));
+        $this->subject->increase('testSourceId', self::DUMMY_ADJUSTMENT_TYPE, 10);
+        $this->assertEquals(10, $this->subject->get('testSourceId'));
+        $this->subject->decrease('testSourceId', self::DUMMY_ADJUSTMENT_TYPE, 5);
+        $this->assertEquals(5, $this->subject->get('testSourceId'));
+        $this->subject->decrease('testSourceId', self::DUMMY_ADJUSTMENT_TYPE, 10);
+        $this->assertEquals(-5, $this->subject->get('testSourceId'));
+    }
+
+    public function testGetByType_ReturnsValuesCalculatedForGIvenType(): void
```

Comment:
Could follow the convention of test functions in the file `test{$functionName}_When{something happened}_Then{expected behavior}`. Otherwise the name is clear.

Reply:
Didn't get it. Will fix.